lib/google_scraping/account/accounts.ex

```diff
@@ -0,0 +1,109 @@
+defmodule GoogleScraping.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias GoogleScraping.Accounts.User
+  alias GoogleScraping.Accounts.UserToken
+  alias GoogleScraping.Accounts.UserTokenQueries
```

Comment:
It can be written as
```suggestion
  alias GoogleScraping.Accounts.{User, UserToken ...}
```

Reply:
Actually, I tried to do so before, and the linter shows an error:

<img width="858" alt="Screen Shot 2022-05-24 at 15 13 09" src="https://user-images.githubusercontent.com/475367/169983902-062eed32-792b-4092-ab9e-ded85cef83ce.png">

Do you suggest suppressing the warning for this line? Or disable the rule itself?

Reply:
Hi @Nihisil, this is actually a known [issue of credo](https://github.com/rrrene/credo/issues/441)

The current workaround is to exclude the offending files that are using the single line directives in `.credo.exs`

```elixir
        {Credo.Check.Consistency.MultiAliasImportRequireUse,
         files: %{
           excluded: [
             "lib/google_scraping_web.ex",
             "test/support/conn_case.ex"
           ]
         }},
```

and adjusting the `test/support/data_case.ex` so that it uses the `multi-alias` syntax. 

I'll create an issue on our elixir template so we can avoid this issue in the future 👍

Reply:
Thanks 🙏